docs/configuration/settings/algorithm/smb-settings.md

```diff
@@ -194,7 +194,7 @@ This safety limiter looks at the difference between your last two blood glucose
         Compare to Max Allowed Glucose Rise for SMB
         
         $$
-        Delta\ Percentage \gt \ or\ = \ or\ \lt Max\ Allowed\ Glucose\ Rise\ for\ SMB\ Setting
+        Rise\ Percentage \gt \ or\ = \ or\ \lt Max\ Allowed\ Glucose\ Rise\ for\ SMB\ Setting
```

Comment:
I suggest using `\mathit{SM}\mathit{B}` instead of `SMB` to prevent the equation from breaking as soon as we will add `SMB` to the glossary.  
This is unfortunately the only workaround I found so far. It makes the equation more cryptic!

See PR #119 for more details.